examples/hashes/connector.py

```diff
@@ -0,0 +1,141 @@
+# This example shows how to calculate hash of fields to be used as primary key. This is useful in scenarios where the
```

Comment:
should we keep it here, or move to one for 3 category wise directories?

Reply:
Moved to `Common patterns for connector` folder